src/main/java/org/terracotta/build/plugins/AbstractKitTestingPlugin.java

```diff
@@ -111,6 +111,10 @@ public void apply(Project project) {
             fcd.exclude();
           }
         });
+        spec.filesMatching("**/*.jar", fcd -> {
```

Comment:
I think what you're doing is okay... but I think the comment isn't correct. We don't know what dependency strategy is going to be used. I think it's safe to exclude here because of what we're building though.

Reply:
Ah! That's right... Did not thing of that!